### DISCUSSION TITLE
Customisability

### DISCUSSION THREAD

**Discussion by BlueFireRO** [pelican-dev/panel#74] [Ideas]
The ability to customise colors of buttons,background,console,and others , the ability to put a photo or video as background by either uploading it or with the link path to the file the ability to set a text or image logo, the ability to add the discord widgetbot(it would be great for hosting companies as they could write in discord support channels right from the panel)

**Comment by BlueFireRO** [comment]
Also in this discussion to place a banner specific to the egg would be cool,like to be able to put a minecraft view on top of servers created with the minecraft egg,something like this:
![image](https://github.com/pelican-dev/panel/assets/64308848/b0c405a0-6fce-4092-b814-fda6f4c5c9e6)

**Comment by lancepioch** [comment]
Thanks, you will be able to do this via plugins and themes for free!
